frontend/mulighetsrommet-veileder-flate/src/apps/modia/ModiaRoute.ts

```diff
@@ -41,6 +46,8 @@ export function resolveRoutePath(route: ModiaRouteParams): string {
       return `/arbeidsmarkedstiltak/deltakelse/${route.gjennomforingId}`;
     case ModiaRoute.ARBEIDSMARKEDSTILTAK_DELTAKELSE:
       return `/arbeidsmarkedstiltak/deltakelse/deltaker/${route.deltakerId}`;
+    case ModiaRoute.ARBEIDSMARKEDSTILTAK_DETALJER:
+      return `/arbeidsmarkedstiltak/tiltak/${route.gjennomforingId}`;
```

Comment:
Disse modia-routene trengs kun når vi lenker til andre microfrontends. Siden dette er en intern lenke i samme app så holder det å bruke en standard react-router Link i stedet for å følge oppsettet med `resolveModiaRoute`